docs/guide/3-install-free5gc.md

```diff
@@ -151,10 +151,10 @@ sudo systemctl disable ufw # prevents the firewall to wake up after a OS reboot
 1. Before building WebConsole, install nodejs and yarn packages first:
 
     ```
-    sudo apt remove cmdtest
-    sudo apt remove yarn
+    sudo apt remove yarn cmdtest
     curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
     echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+    curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash - 
```

Comment:
The Node.js 20 is the latest LTS version and works fine with [free5gc/webconsole](https://github.com/free5gc/webconsole).

```suggestion
    curl -fsSL https://deb.nodesource.com/setup_20.x | sudo -E bash - 
```